src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -144,9 +157,9 @@ public function setPlatform($platform)
      *
      * @return \Propel\Generator\Platform\PlatformInterface
      */
-    public function getPlatform()
+    public function getPlatform(): PlatformInterface
     {
-        if ($this->platform === null) {
+        if (!isset($this->platform)) {
```

Comment:
What's the reason behind this change? `!isset` is slower than `=== null`

Reply:
I would also like to avoid seeing any unrelated change here that is not necessary as it only blocks moving forward.

Reply:
Starting from PHP 7.4, when you use strict types, the properties aren't initialized to null anymore, so  testing `=== null` rises a `Fatal error: Uncaught Error: Typed property QuickBuilder::$platform must not be accessed before initialization`.

Anyway, I can explicitly initialize the variables to `null`, if you think it's better.

Reply:
I adjusted the code to change it as little as possible.